internal/plugin/connectors/tcp/mssql/connector_test.go

```diff
@@ -2,22 +2,24 @@ package mssql
 
 import (
 	"context"
+	"errors"
+	"io"
 	"net"
 	"testing"
 
 	"github.com/stretchr/testify/assert"
 
 	"github.com/cyberark/secretless-broker/internal/plugin/connectors/tcp/mssql/mock"
 	logmock "github.com/cyberark/secretless-broker/pkg/secretless/log/mock"
-	pluginconnector "github.com/cyberark/secretless-broker/pkg/secretless/plugin/connector"
+	"github.com/cyberark/secretless-broker/pkg/secretless/plugin/connector"
 	"github.com/denisenkom/go-mssqldb"
 )
 
 func TestHappyPath(t *testing.T) {
 	logger := logmock.NewLogger()
 	clientConn := mock.NewNetConn(nil)
 	expectedBackendConn := mock.NewNetConn(nil)
-	creds := pluginconnector.CredentialValuesByID{
+	creds := connector.CredentialValuesByID{
```

Comment:
best to name the import otherwise it conflicts with the connector variable below